pagopa/io-web-profile
# [IOPID-919] : Logout Flow Adjustment

## Short Description
Updated OpenAPI with the latest BFF spec file.

## List of Changes Proposed in This Pull Request
- Logout Adjustment (204 instead of 200)
## How to Test
Log in to the portal
Initiate the "logout" flow
The logout API should respond with a status code of 204 instead of 200.
Ensure it follows the success path.
